zyte_common_items/items.py

```diff
@@ -612,3 +613,94 @@ class BusinessPlace(Item):
 
     #: Data extraction process metadata.
     metadata: Optional[BusinessPlaceMetadata] = None
+
+
+@attrs.define(slots=True, kw_only=True)
+class RealEstateListing(Item):
+    #: The url of the final response, after any redirects.
+    url: str = attrs.field(converter=url_to_str)
+
+    #: Webpage `breadcrumb trail`_.
+    #:
+    #: .. _Breadcrumb trail: https://en.wikipedia.org/wiki/Breadcrumb_navigation
+    breadcrumbs: Optional[List[Breadcrumb]] = None
+
+    #: The identifier of the real estate, usually assigned by the seller and unique within a website, similar to product SKU.
+    realEstateId: Optional[str] = None
+
+    #: The name of the real estate.
+    name: Optional[str] = None
+
+    #: Publication date of the real estate offer.
+    #: Format: ISO 8601 format: "YYYY-MM-DDThh:mm:ssZ"
+    #: With timezone, if available.
```

Comment:
Comments need to be written in reStructuredText, which needs empty lines to make paragraphs. Line breaks are otherwise stripped during rendering.

This applies to other comments as well. I recommend you build the docs with `tox -e docs` and check how the documentation is rendered.

```suggestion
    #: Publication date of the real estate offer.
    #:
    #: Format: ISO 8601 format: "YYYY-MM-DDThh:mm:ssZ"
    #:
    #: With timezone, if available.
```